src/containers/feedOfClaim/index.tsx

```diff
@@ -142,44 +91,45 @@ const FeedClaim = ({ toggleSnackbar, setSnackbarMessage, setLoading }: IHomeProp
         flexDirection: 'column'
       }}
     >
-      {claims.map((claim, index) => (
-        <div key={claim.id}>
-          <Card
-            sx={{
-              maxWidth: 'fit',
-              height: 'fit',
-              m: '20px',
-              borderRadius: '20px',
-              border: '2px solid #009688',
-              display: 'flex'
-            }}
-          >
-            <CardContent>
-              <Typography sx={{ padding: '5px 0 0 5px' }}>{formatClaimText(claim)}</Typography>
-            </CardContent>
+      {claims &&
```

Comment:
If like previous version, you initialized `claims` with `[]`, you wouldn't  have to check if the claims are truthy or not.